ddpui/celeryworkers/tasks.py

```
@@ -1069,6 +1071,57 @@ def check_for_long_running_flow_runs():
+    if not jobs:
+        logger.error(f"No jobs found for job_id={job_id}")
+        return None
+
+    job_data: dict = jobs[0].get("job", {})
+    attempts: list[dict] = jobs[0].get("attempts", [])
+    if not attempts:
+        logger.info(f"No attempts found for job_id={job_id}")
+        latest_attempt = None
+    else:
+        # Pick the attempt with the greatest id
+        latest_attempt = max(attempts, key=lambda a: a.get("id", 0))
```

Ruff (0.11.9)

Local variable `latest_attempt` is assigned to but never used

Remove assignment to unused variable `latest_attempt`

(F841)